1.8/overview/concepts.md

```diff
@@ -1,121 +1,282 @@
 
-# <a name="dcosmarathon"></a>DC/OS Marathon
+##### <a name="private-agent-node"></a>Private Agent Node
 
-The native Marathon instance that is the “init system” for DC/OS. It starts and monitors DC/OS applications and services.
+A private agent node is an agent node that is on a network that** ***does not* have ingress access from outside of the cluster via the cluster’s infrastructure networking.
 
-# <a name="dcosservice"></a>DC/OS service
+- The Mesos agent on each private agent node is (by default) configured with none of its resources allocated to any specific Mesos roles (`*`).
+- Most service packages install by default on private agent nodes.
+- Clusters are usually made up of mostly private agent nodes.
 
-DC/OS services are applications that are packaged and available from the public [GitHub package repositories][5]. Available DC/OS services include Mesos frameworks and other applications.
+##### <a name="public-agent-node"></a>Public Agent Node
 
-# <a name="executor"></a>Executor
+A public agent node is an agent node that is on a network that *does* have ingress access from outside of the cluster via the cluster’s infrastructure networking.
 
-A framework running on top of Mesos consists of two components: a scheduler that registers with the master to be offered resources, and an executor process that is launched on agent nodes to run the framework’s tasks. For more information about framework schedulers and executors, see the [App/Framework development guide][6].
+- The Mesos agent on each public agent node is configured with the "public_ip:true" attribute and all of its resources allocated to the `slave_public` role (`agent_public` in the future).
+- Public agent nodes are used primarily for externally facing reverse proxy load balancers, like Marathon-LB.
+- Clusters usually only have a few public agent nodes, because a single load balancer can handle proxying multiple services.
+- For more information, see [Converting Agent Node Types](/docs/1.8/administration/installing/custom/convert-agent-type/).
 
-# <a name="exhibitorforzookeeper"></a>Exhibitor for ZooKeeper
+### <a name="bootstrap-machine"></a>Bootstrap Machine
 
-DC/OS uses ZooKeeper, a high-performance coordination service to manage the installed DC/OS services. Exhibitor automatically configures your ZooKeeper installation on the master nodes during your DC/OS installation.
+A bootstrap machine is the machine on which the DC/OS installer artifacts are configured, built, and distributed.
 
-# <a name="framework"></a>Framework
+- The bootstrap machine is not technically considered part of the cluster, in that it does not have DC/OS installed on it (this may change in the future), but it does need to be accessible to/from (depending on install method) the machines in the cluster via infrastructure networking in order to perform remote installation.
```

Comment:
- `part of the cluster, in that it does not` to `part of the cluster since it does not`
- New sentence: `..., but it does need to...` to `The bootstrap node must be accessible to and from the machines (depending on install method) in the cluster via infrastructure networking to perform remote installation.`